mbus-api/protobuf/v1/event.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+import "google/protobuf/timestamp.proto";
+
+package v1.event;
+
+// Event Message
+message EventMessage {
+    // Event category
+    EventCategory category = 1;
+    // Event action
+    EventAction action = 2;
+    // Target id for the category against which action is performed
+    string target = 3;
+    // Event meta data
+    EventMeta metadata = 4;
+}
+
+// Event Category
+enum EventCategory {
```

Comment:
Ah so here is Category, so perhaps my suggestion of Resource is not correct @sinhaashish ..

Reply:
Let @datacore-vvarakantham  PR get in, will update my changes accordingly , There are few renames done in her PR